sdk/src/wallet/mod.rs

```diff
@@ -9,7 +9,7 @@ pub mod account;
 /// The core module.
 pub mod core;
 #[cfg(any(feature = "stronghold", feature = "storage"))]
-pub(crate) mod migration;
+pub mod migration;
```

Comment:
Why is migration public?

Reply:
some types are, like the version struct

Reply:
And out of curiosity why is the version struct public? Seems pretty self contained to me

Reply:
Just seems like something users may want to be able to deserialize themselves.

Reply:
IMO users shouldn't even be aware of migrations but alright.

Reply:
yea i agree, all migration related code should be internal only.
as a user id never want to touch it :D